var/spack/repos/builtin/packages/python/package.py

```diff
@@ -542,6 +542,10 @@ def configure_args(self):
         config_args = []
         cflags = []
 
+        # https://bugs.python.org/issue1294959
+        if "fedora" in self.spec.os or "suse" in self.spec.os:
```

Comment:
is it possible to do this without `spec.os`?

Reply:
Lib is already the default but doesn't seem to work on Fedora/SUSE, I think it needs to be lib64. We can do it without `spec.os`, I just need some way of checking to see which distro we're on.

Reply:
What makes it such that it needs to be lib64? Something in Python itself? Where?

Reply:
No idea, try reading through https://bugs.python.org/issue1294959 and see if you can extract more info than me.